.circleci/config.yml

```diff
@@ -75,7 +90,10 @@ jobs:
               sed -i "s/version '.*'/version '${CIRCLE_TAG:1}'/" build.gradle
               awk '/version '\''/' build.gradle
             fi
+            export LOCAL_JVM=true
```

Comment:
I think it better to use circleci environment values instead of export command here

Reply:
@whoisxx 
I removed this statement from config at this [commit](https://github.com/klaytn/caver-java/pull/240/commits/466248cff26478e2ebb20ad0c5c2c7fe7ae1b62f).
So it's okay not to check this part.

Thanks for your feedback.

If you want to know the reason why I used this environment variable, please read the below.
-------------
What I tried is to declare environment variable in Android Emulator and skip some tests based on this, but couldn't find about declaring env variable in Android emulator when testing.

So I did reversely. Declare env variable at host machine(Ubuntu).
But if we accept this idea, we must set env variable `LOCAL_JVM` every time we test on JVM. That's quite cumbersome and difficult to remember.

Reply:
Understand. thank you for explanation